### ISSUE TITLE
On Travis-CI, CMake is finding JDK 13 rather than JDK 8

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#2646]
Travis-CI builds are failing because CMake is finding JDK 13, but we need JDK 8:

```
-- Found Java: /Library/Java/JavaVirtualMachines/openjdk-13.0.1.jdk/Contents/Home/bin/java (found suitable version "13.0.1", minimum required is "1.7") 
```

**Comment by jenhicks** [comment]
What's the status of this one @chrisdembia ?

**Comment by chrisdembia** [comment]
This can be closed due to #2652.
